cluster-autoscaler/core/scaledown/actuation/actuator.go

```diff
@@ -346,10 +349,16 @@ func (a *Actuator) deleteNodesAsync(nodes []*apiv1.Node, nodeGroup cloudprovider
 
 		if force {
 			go a.nodeDeletionScheduler.scheduleForceDeletion(nodeInfo, nodeGroup, batchSize, drain)
+			if a.nodeLatencyTracker != nil {
+				a.nodeLatencyTracker.ObserveDeletion(node.Name, time.Now())
```

Comment:
`ObserveDeletion` -> `ObserveDeletionStart`? The actual deletion doesn't complete at the moment this function is called.